File: tddphp/TestMatematicas.php
<?php
require 'Matematicas.php';
class TestMatematicas extends PHPUnit_Framework_TestCase
{
	public function testAdd() {
		$num1=3;
		$num2=2;
		$result=5;
		$this->assertEquals(Matematicas.suma(num1, num2), result);
		
	}
}

File: tddjava/src/tddjava/TestMatematicas.java
package tddjava;

import static org.junit.Assert.*;

import org.junit.Test;

public class TestMatematicas {

	@Test
	public void testAdd() {
		int num1=3;
		int num2=2;
		int result=6;
		assertEquals(Matematicas.suma(num1, num2), result);
		
	}

}
